apis/python/src/tiledb/vector_search/ingestion.py

```diff
@@ -2520,21 +2688,30 @@ def consolidate_and_vacuum(
         group = tiledb.Group(index_group_uri, "r")
         temp_size = int(group.meta.get("temp_size", "0"))
         group.close()
-        group = tiledb.Group(index_group_uri, "w")
-        if index_timestamp is None:
-            index_timestamp = int(time.time() * 1000)
-        ingestion_timestamps.append(index_timestamp)
-        base_sizes.append(temp_size)
-        partition_history.append(partitions)
-        group.meta["partition_history"] = json.dumps(partition_history)
-        group.meta["base_sizes"] = json.dumps(base_sizes)
-        group.meta["ingestion_timestamps"] = json.dumps(ingestion_timestamps)
-        group.close()
+        
+        if index_type != "VAMANA":
```

Comment:
We can introduce some util functions to check if an algorithm is `type-erased` in order to perform the logic here and above?

Reply:
Thanks, good call, updated.